test/openAPI/features/support/helpers/helpers.js

```diff
@@ -1,4 +1,41 @@
 module.exports = {
   localhost: 'http://localhost:3333/',
   defaultResponseTime: 15000,
+  sendSingleEmailEndpoint: 'send/email/single',
+  contentTypeHeader: {
+    key: 'content-type',
+    value: 'application/json; charset=utf-8',
+  },
+  sendEmailSingleResponseSchema: {
+    type: 'object',
+    properties: {
+      response: { type: 'string' },
+      requestUID: { typep: 'string' },
+    },
+  },
```

Comment:
The same schema will need to be use for `/send/email/batch` so I propose change the name to `responseRequestAcceptedSchema` how it is in API spec. And then write in the same file `sendEmailSingleResponseSchema: this.responseRequestAcceptedSchema` or just use `responseRequestAcceptedSchema`

Reply:
Changed to ```sendEmailResponseSchema``` so it will be used for both single and batch endpoints